py_gtfs_rt_ingestion/ingest.py

```diff
@@ -111,11 +110,12 @@ def main() -> None:
     """every second run jobs that are currently pending"""
     # start rds writer process
     # this will create only one rds engine while app is running
-    metadata_queue, _rds_process = start_rds_writer_process()
+    metadata_queue, rds_process = start_rds_writer_process()
 
     schedule.every(5).minutes.do(ingest, metadata_queue=metadata_queue)
 
     while True:
+        check_for_sigterm(metadata_queue, rds_process)
         schedule.run_pending()
         time.sleep(1)
```

Comment:
this seems like a better spot for `check_for_sigterm` function, is continuously checked whenever `ingest` is not running.